benchmarks/dynamo/common.py

```diff
@@ -2131,6 +2135,12 @@ def cast_based_on_args(self, model, example_inputs):
                 )
                 self.args.amp = True
                 self.setup_amp()
+            elif self.args.only in self.force_fp16_for_bf16_models:
+                log.warning(
+                    "Model %s does not support float16, running with bfloat16 instead",
```

Comment:
```suggestion
                    "Model %s does not support bfloat16, running with float16 instead",
```

Reply:
also, when should we force to fp16 and when should we force to amp (code above this condition)

Reply:
Sometimes models need more precision than fp16/bf16, so we run in amp. In this case the kernel does not support bf16